src/main/java/net/sf/jabref/logic/CustomEntryTypesManager.java

```diff
@@ -1,30 +1,28 @@
 package net.sf.jabref.logic;
 
-import java.util.ArrayList;
-import java.util.Iterator;
 import java.util.List;
-import java.util.Optional;
+import java.util.stream.Collectors;
 
 import net.sf.jabref.model.EntryTypes;
 import net.sf.jabref.model.database.BibDatabaseMode;
 import net.sf.jabref.model.entry.CustomEntryType;
-import net.sf.jabref.model.entry.EntryType;
 import net.sf.jabref.preferences.JabRefPreferences;
 
 public class CustomEntryTypesManager {
 
-    public static final List<EntryType> ALL = new ArrayList<>();
     /**
      * Load all custom entry types from preferences. This method is
      * called from JabRef when the program starts.
      */
     public static void loadCustomEntryTypes(JabRefPreferences prefs) {
```

Comment:
Mhmmm... This is basically just a UI-Helper class. `loadCustomEntryTypes` is called from the JabRefMain and the CLI Argument processer.

Storage is also used in the UI only...

So would it be possible and sensible to move it as is into the `gui` package?